src/actions/_common-action-utils.js

```diff
@@ -49,3 +49,21 @@ export const throwStringifiedError = err => {
 		throw err.message || JSON.stringify(err);
 	}
 };
+
+export async function getTransformedTemplate(template, data, cfg) {
+	// transform was already typechecked in interface check
```

Comment:
We might want to update this comment to be clear that this type-checking is done at the runtime level